llvm/include/llvm/TableGen/Record.h

```diff
@@ -1798,9 +1793,9 @@ class Record {
   }
 
   void removeValue(const Init *Name) {
-    for (unsigned i = 0, e = Values.size(); i != e; ++i)
-      if (Values[i].getNameInit() == Name) {
-        Values.erase(Values.begin()+i);
+    for (auto [Idx, Value] : enumerate(Values))
```

Comment:
Can we use `find_if` and pass the iterator to `erase`?

Reply:
Done. thanks